app/views/hr_view.py

```diff
@@ -56,6 +56,8 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
   line_manager = format_line_manager(current_job_role)
 
+  employee_info['address'] = map_full_address_details(employee_info)
```

Comment:
Ditto, as with the above comment on input mutation.

Reply:
This hasn't been resolved in the past commits.

Reply:
removed